service/src/main/java/uk/gov/mca/beacons/api/export/application/ExportService.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.mca.beacons.api.export.application;
+
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.mca.beacons.api.accountholder.application.AccountHolderService;
+import uk.gov.mca.beacons.api.accountholder.domain.AccountHolder;
+import uk.gov.mca.beacons.api.beacon.application.BeaconService;
+import uk.gov.mca.beacons.api.beacon.domain.BeaconId;
+import uk.gov.mca.beacons.api.exceptions.ResourceNotFoundException;
+import uk.gov.mca.beacons.api.export.mappers.ExportMapper;
+import uk.gov.mca.beacons.api.export.rest.BeaconExportDTO;
+import uk.gov.mca.beacons.api.export.rest.LabelDTO;
+import uk.gov.mca.beacons.api.legacybeacon.application.LegacyBeaconService;
+import uk.gov.mca.beacons.api.legacybeacon.domain.LegacyBeacon;
+import uk.gov.mca.beacons.api.legacybeacon.domain.LegacyBeaconId;
+import uk.gov.mca.beacons.api.note.application.NoteService;
+import uk.gov.mca.beacons.api.registration.application.RegistrationService;
+import uk.gov.mca.beacons.api.registration.domain.Registration;
+
+@Service("ExportService")
+public class ExportService {
+
+  private final BeaconService beaconService;
+  private final RegistrationService registrationService;
+  private final LegacyBeaconService legacyBeaconService;
+  private final ExportMapper exportMapper;
+  private final NoteService noteService;
+  private final AccountHolderService accountHolderService;
+
+  @Autowired
+  public ExportService(
+    BeaconService beaconService,
+    RegistrationService registrationService,
+    LegacyBeaconService legacyBeaconService,
+    ExportMapper exportMapper,
+    NoteService ns,
+    AccountHolderService accountHolderService
+  ) {
+    this.beaconService = beaconService;
+    this.registrationService = registrationService;
+    this.legacyBeaconService = legacyBeaconService;
+    this.exportMapper = exportMapper;
+    this.noteService = ns;
+    this.accountHolderService = accountHolderService;
+  }
+
+  public BeaconExportDTO getBeaconExportDTO(
+    UUID rawBeaconId,
+    String noteGeneratedType
```

Comment:
So we're passing in a noteGeneratedType and that becomes part of the text for the note. Nice